ibmmq/README.md

```diff
@@ -0,0 +1,23 @@
+# MQ connection test tool
+
+The purpose of the IBM MQ connection test tool is to test whether IBM MQ is configured and can be connected. You can select any of the following connection modes:
+
+1. Local binding mode: Provide the `IBM MQ lib` path parameter. You can run the IBM MQ test tool on the same server where the IBM MQ runs.
+
+2. Client binding mode: Provide the channel name and channel port to use the IBM MQ connection. If the security is enabled for the IBM MQ channel, then you need a username and password to log in to the channel. By using client binding mode, you can run the IBM MQ test tool on another server or on the same server where the IBM MQ runs. 
+
+## Usage of the IBM MQ connection test tool
+
+- You need to export JAVA_HOME before the tool is invoked.
```

Comment:
```suggestion
- You need to export JAVA_HOME before you use the IBM MQ connection test tool.
```

Reply:
`JAVA_HOME` is an environment variable. It's not a file. So I'll modify this sentence as
```suggestion
- You need to export the `JAVA_HOME` before you use the IBM MQ connection test tool.
```